proto/v1/skaffold.pb.go

```diff
@@ -1,11 +1,12 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: skaffold.proto
+// source: v1/skaffold.proto
 
 package proto
 
 import (
 	context "context"
 	fmt "fmt"
+	enums "github.com/GoogleContainerTools/skaffold/proto/enums"
```

Comment:
Can this be sorted to preserve the Skaffold style of placing Skaffold imports at the end?